doc/additional_tools/pyreverse/index.rst

```diff
@@ -10,63 +11,8 @@ If Graphviz (or the ``dot`` command) is installed, all `output formats supported
 can be used as well. In this case, ``pyreverse`` first generates a temporary ``.gv`` file, which is then
 fed to Graphviz to generate the final image.
 
-Running Pyreverse
-'''''''''''''''''
-
-To run ``pyreverse``, use::
-
-  pyreverse [options] <packages>
-
-<packages> can also be a single Python module.
-To see a full list of the available options, run::
-
-   pyreverse -h
-
-Example Output
-''''''''''''''
-
-Example diagrams generated with the ``.puml`` output format are shown below.
-
-Class Diagram
-.............
-
-.. image:: ../../media/pyreverse_example_classes.png
-   :width: 625
-   :height: 589
-   :alt: Class diagram generated by pyreverse
-   :align: center
-
-
-Package Diagram
-...............
-
-.. image:: ../../media/pyreverse_example_packages.png
-   :width: 344
-   :height: 177
-   :alt: Package diagram generated by pyreverse
-   :align: center
-
-
-Creating Class Diagrams for Specific Classes
-''''''''''''''''''''''''''''''''''''''''''''
-
-In many cases creating a single diagram depicting all classes in the project yields a rather unwieldy, giant diagram.
-While limiting the input path to a single package or module can already help greatly to narrow down the scope, the ``-c`` option
-provides another way to create a class diagram focusing on a single class and its collaborators.
-For example, running::
-
-  pyreverse -ASmy -c pylint.checkers.classes.ClassChecker pylint
-
-will generate the full class and package diagrams for ``pylint``, but will additionally generate a file ``pylint.checkers.classes.ClassChecker.dot``:
-
-.. image:: ../../media/ClassChecker_diagram.png
-   :width: 757
-   :height: 1452
-   :alt: Package diagram generated by pyreverse
-   :align: center
-
 .. toctree::
   :maxdepth: 1
```

Comment:
Shouldn't we tweak this?

Reply:
I'm trying to currently, we managed to do it for the dev guide so I should be able to.

Reply:
This is surprisingly hard. Might have to change the headings underline again.

Reply:
I have a possible solution that fixes both the heading hierarchy and improves the documentation readability. Would you like me to create a commit with these changes?

Reply:
Sure @Julfried, thank you for proposing (I tried to increase maxdepth without success)

Reply:
Here is my idea. One thing I noticed: this table of contents collapses when the window is narrow on smaller screens. Maybe this could be fixed in the theme settings. What do you think?